src/Template/Pages/downloads.ctp

```diff
@@ -146,6 +146,9 @@ $this->set('title_for_layout', $this->Pages->formatTitle(__('Download sentences'
         $license_str = __('License');
         $attribution_url_str = __('Attribution URL');
         $tab_str = __('tab');
+        $per_language_str = __('Per-language files are available at <a ' .
+            'href="https://downloads.tatoeba.org/exports/per_language/">' .
+            'https://downloads.tatoeba.org/exports/per_language</a>.');
```

Comment:
There are two problems with this translatable string.

1. It includes a URL, however the URL shouldn’t be showed to translators because they may modify it by mistake. Instead, use:
```php
$per_language_str = format(
    __('Here is my <a href="{}">link</a>.'),
    'https://downloads.tatoeba.org/exports/per_language'
);
```

2. The [hyperlink text is not good](https://agsci.psu.edu/communications/how-to/websites/web-best-practices/appropriate-link-text). I know that other existing links on that page are like that too, but feel free to make it right.

Reply:
> I know that other existing links on that page are like that too, but feel free to make it right.

How about something like this:
![downloads](https://user-images.githubusercontent.com/6566321/75660772-7527fe00-5c6c-11ea-9ff9-7236a4c64cb2.png)

Reply:
You could have a dropdown to select the language and have a download button.

![Screenshot_2020-03-03 Download sentences - Tatoeba](https://user-images.githubusercontent.com/221850/75809596-51a6a580-5d89-11ea-8ecf-040e6243859b.png)

For the files where there's no possibility to download in a specific language, you just don't have the dropdown.

Reply:
The language drop down seems like a better idea. I think it’s important however to let people do "right-click → save as" or "right-click → copy link URL", so the download button should allow to do so. I think you could add a checkbox that makes the language dropdown selectable, like this:

> [✘] Only include sentences in: [Select a language]
> [Download]

This way, it makes it easier to understand that the language dropdown is a filtering option.

@AndiPersti Or, if you don’t feel like doing all that frontend stuff, make it simple:

> <h3>Sentences</h3>
>
> <b>Filename</b>
> sentences.csv
>
> <b>Download</b>
> Download this file <a href="https://downloads.tatoeba.org/exports/sentences.tar.bz2">including all languages</a> or <a href="https://downloads.tatoeba.org/exports/per_language/">only for a single language</a>.

Or

> Download this file <a href="http://example.com/">including all languages</a>.